live/lib/auth_app_web/live/user_live/login.ex

```diff
@@ -1,49 +1,128 @@
 defmodule AuthAppWeb.UserLive.Login do
   use AuthAppWeb, :live_view
 
+  alias AuthApp.Accounts
+
   def render(assigns) do
     ~H"""
     <div class="mx-auto max-w-sm">
       <.header class="text-center">
-        Log in to account
-        <:subtitle>
-          Don't have an account?
-          <.link navigate={~p"/users/register"} class="font-semibold text-brand hover:underline">
-            Sign up
-          </.link>
-          for an account now.
+        <p :if={!@current_user}>Log in</p>
+        <p :if={@current_user}>Log in to re-authenticate</p>
+        <:subtitle :if={!@current_user}>
+          Don't have an account? <.link
+            navigate={~p"/users/register"}
+            class="font-semibold text-brand hover:underline"
+            phx-no-format
+          >Sign up</.link> for an account now.
         </:subtitle>
       </.header>
 
-      <.simple_form for={@form} id="login_form" action={~p"/users/log-in"} phx-update="ignore">
-        <.input field={@form[:email]} type="email" label="Email" autocomplete="username" required />
+      <.simple_form
+        :let={f}
+        for={@form}
+        id="login_form_magic"
+        action={~p"/users/log-in"}
+        phx-submit="submit_magic"
+      >
+        <.input
+          disabled={!!@current_user}
+          field={f[:email]}
+          type="email"
+          label="Email"
+          autocomplete="username"
```

Comment:
```suggestion
          autocomplete="username"
```

Is username correct?